src/user/dto/user.dto.ts

```diff
@@ -30,8 +32,8 @@ export class UserDto {
   @Matches(password_match, { message: password_message[1] })
   password: string;
 
-  @ApiProperty({ example: false })
-  isAdmin: boolean;
+  @ApiProperty({ type: 'enum', enum: RoleTypes, isArray: true, example: [RoleTypes.User] })
+  roles: RoleTypes[];
```

Comment:
This property shoulden't be here - in this case everyone can send request with Admin role :)